src/test/java/com/reviews/demo/mapper/UserMapperTest.java

```diff
@@ -0,0 +1,94 @@
+package com.reviews.demo.mapper;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+import com.reviews.demo.dto.CommentDto;
+import com.reviews.demo.model.Role;
+import com.reviews.demo.model.User;
+import com.reviews.demo.service.RoleService;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+class UserMapperTest {
+    private static final CommentDto FULL_COMMENT_DTO = CommentDto.builder()
```

Comment:
```suggestion
    private static final CommentDto fullCommentDto = CommentDto.builder()
```
it's not a constant, cause your class is not immutable